golang/05-empty-volume/volume.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"path"
+)
+
+var mounted = func(file string) string { return path.Join("mnt", file) }
+
+type hello struct {
+	Message string `json:"message"`
+}
+
+func main() {
+	var err error
+	// read-write files
+	for i := 0; i < 2; i++ {
+		readdir("mnt", i)
+		// on 1st try, non-existing, should err
+		err = readtxt(mounted("hello.txt"))
+		if err != nil {
+			fmt.Println(err)
+			continue
+		}
+		// write file
+		err = writetxt(mounted("hello.txt"), "lol")
```

Comment:
initially `hello.txt` contains the text `hello`, but replacing it with another text via `ioutil.WrifeFile` does not completely replace the text if the initial content is longer e.g expected new text to be `lol` but got `lollo` instead.

Reply:
i'm unable to reproduce...